app/javascript/components/explore/ExploreView.jsx

```diff
@@ -28,6 +28,10 @@ function RoutableExploreTab({ studyAccession }) {
   async function loadStudyData() {
     const exploreResponse = await fetchExplore(studyAccession)
     setExploreInfo(exploreResponse)
+    // set window.SCP.isDEEnabled so that we can track DE visibility globally
```

Comment:
For clarity could the comment say "so that we track Differential Expression (DE).." just so new folks can be sure what DE stands for